README.md

```diff
@@ -135,6 +134,14 @@ Our current version number has 3 parts. **ex: 2.19.0**
 
 
 
+### Getting Started
```

Comment:
This is great.  We should have had this forever.

Reply:
Can we make these links to the source files?

Reply:
We probably want to do a sweep and update the examples to use Path/other new conventions.  Pointing them out is a good start though.